code/game/objects/items/weapons/cigs_lighters.dm

```diff
@@ -331,8 +333,39 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 	item_state = "luckyoff"
 	icon_on = "luckyon"
 	icon_off = "luckyoff"
+	spawn_tags = SPAWN_TAG_CIGARETTE_MAINTS
 	type_butt = /obj/item/trash/cigbutt/lucky
 
+/obj/item/clothing/mask/smokable/cigarette/robusto
+	desc = "A roll of tobacco and nicotine. Gives the feeling of old wars. Let's crack some skulls!"
+	preloaded_reagents = list("nicotine" = 6, "violence" = 3)
+	icon_state = "robustoff"
+	item_state = "robustoff"
+	icon_on = "robuston"
+	icon_off = "robustoff"
+	spawn_tags = SPAWN_TAG_CIGARETTE_MAINTS
+	type_butt = /obj/item/trash/cigbutt/robusto
+
+/obj/item/clothing/mask/smokable/cigarette/buckshot
+	desc = "A roll of tobacco and nicotine. Maint cig, some say it helps take the edge off better than most."
+	preloaded_reagents = list("nicotine" = 6, "paracetamol" = 3)
+	icon_state = "klassikaoff"
+	item_state = "klassikaoff"
+	icon_on = "klassikaon"
+	icon_off = "klassikaoff"
+	spawn_tags = SPAWN_TAG_CIGARETTE_MAINTS
+	type_butt = /obj/item/trash/cigbutt/buckshot
+
+/obj/item/clothing/mask/smokable/cigarette/quasimorph
+	desc = "A roll of tobacco and nicotine. Maint cig, ancient paper, how old is this...?"
+	preloaded_reagents = list("nicotine" = 6, "dexalin" = 3)
+	icon_state = "kosmosoff"
+	item_state = "kosmosoff"
+	icon_on = "kosmoson"
+	icon_off = "kosmossoff"
```

Comment:
```suggestion
	icon_off = "kosmosoff"
```
I only see `kosmosoff`, are you sure this sprite appears? Please double check.
Rest should be ok.

Reply:
![image](https://github.com/discordia-space/CEV-Eris/assets/57843385/e39b7f2d-cb20-49dc-bf5e-593e9e252acf)
yep